sdk/core/Azure.Core.TestFramework/src/ProxyTransport.cs

```diff
@@ -34,6 +35,7 @@ public ProxyTransport(TestProxy proxy, HttpPipelineTransport transport, TestReco
             _filter = filter;
 
             bool useFiddler = TestEnvironment.EnableFiddler;
+            _useDefaultClientIDFormat = TestEnvironment.DefaultClientGuidFormatInRecording;
```

Comment:
I don't think this work in playback during CI because the environment variable won't be set. Instead we can add a property to RecordedTestBase, pass it through to TestRecording, and then read from the _recording variable here.

Reply:
Thanks, I updated to the code to get the variable in RecordedTestBase and save it to the variables